templates/search/index.html.twig

```diff
@@ -68,7 +68,7 @@
             </div>
         </div>
     {% else %}
-            {{ 'search_no_results'|trans({'%searchTerm%': search}, 'search') }}
+            {{ 'No results for %search%'|trans({'%searchTerm%': search}, 'search') }}
```

Comment:
There's something wrong here: You use `%search%` as placeholder and replace`%searchTerm%`.

Reply:
Fixed in 32582b7d6b8c5be72f347f8f674ae75334c57b7d.